src/main/java/pl/futurecollars/invoicing/App.java

```diff
@@ -1,26 +1,15 @@
 package pl.futurecollars.invoicing;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
-import java.io.File;
-import java.io.IOException;
-import java.math.BigDecimal;
-import java.time.LocalDate;
-import java.util.List;
-import pl.futurecollars.invoicing.db.Database;
-import pl.futurecollars.invoicing.db.memory.InMemoryDatabase;
-import pl.futurecollars.invoicing.model.Company;
-import pl.futurecollars.invoicing.model.Invoice;
-import pl.futurecollars.invoicing.model.InvoiceEntry;
-import pl.futurecollars.invoicing.model.Vat;
-import pl.futurecollars.invoicing.service.InvoiceService;
-import pl.futurecollars.invoicing.utils.FilesService;
-import pl.futurecollars.invoicing.utils.JsonService;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@SpringBootApplication
 public class App {
 
-    public static void main(String[] args) throws IOException {
-        Database db = new InMemoryDatabase();
+    public static void main(String[] args) {
+        SpringApplication.run(App.class, args);
+    }
+    /* Database db = new InMemoryDatabase();
```

Comment:
Nie zostawiaj takich komentarzy w kodzie, nawet jak może się przydać później to historię masz w gicie, a potem często taki kod jest zapominany i niepotrzebnie śmieci